src/recensio/plone/behaviors/editorial.py

```diff
@@ -6,10 +6,13 @@
 from plone.supermodel import model
 from recensio.plone import _
 from recensio.plone.behaviors.directives import fieldset_reviewed_text
+from recensio.plone.utils import getFormatter
 from z3c.relationfield.schema import RelationChoice
 from z3c.relationfield.schema import RelationList
 from zope import schema
 from zope.component import adapter
+from zope.i18n import translate
```

Comment:
Should we go for `plone.api.portal.translate`?

Reply:
`api.portal.translate` expects a msgid string. Can I also translate a `Message` object with it?

Reply:
I am confused about this question especially because I do not know what a "msgid string" is.

Isn't the `plone.recensio._` message factory returning a `Message` instance?

I think that:
```
api.portal.translate(_("foo"))
```
and
```
api.portal.translate(Message("foo", "recensio"))
```
are the same exact thing.

I gave that for granted but I might be wrong.

Reply:
Never mind, I've figured it out.